modules/nixos/conduwuit.nix

```
@@ -0,0 +1,267 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.chaotic.conduwuit;
+  defaultUser = "conduwuit";
+  defaultGroup = "conduwuit";
+
+  format = pkgs.formats.toml { };
+  configFile = format.generate "conduwuit.toml" cfg.settings;
+in
+{
+  meta.maintainers = with lib.maintainers; [ s0me1newithhand7s ]; # for now, at least
+  options.chaotic.conduwuit = {
+    enable = lib.mkEnableOption ''
+      Enables conduwuit service.
+    '';
```

```suggestion
    enable = lib.mkEnableOption "conduwuit service";
```

This is expanded to `Whether to enable conduwuit service.`